nginx.conf

```diff
@@ -62,9 +153,7 @@ http {
     }
 
     location / {
-      # Specify the dist/ directory of the repository to `nginx -p` so that . is
-      # relative to dist/.
-      root .;
+      root ../dist;
```

Comment:
Now running from `tmp/`.  Could also run from the project root, and then explicitly put logs in `tmp/`.

Reply:
Is `tmp/` just for storing logs? For better or for worse, I'm actually not sure that I've ever used the logs from this local nginx.

Reply:
It stores the access log, and the nginx pid file.  I think ideally the pidfile wouldn't be necessary, and the logs would go to stdout...

Reply:
Ref pidfile, there isn't much in the nginx docs: https://nginx.org/en/docs/ngx_core_module.html#pid

This thread looks relevant, but doesn't suggest there's a reliable & convenient alternative: https://github.com/docker-library/httpd/issues/166